ArduPlane/AP_ExternalControl_Plane.cpp

```diff
@@ -38,4 +38,18 @@ bool AP_ExternalControl_Plane::set_airspeed(const float airspeed)
 #endif
 }
 
+/*
+    Request gliding
+*/
+bool AP_ExternalControl_Plane::request_gliding(bool gliding_requested)
+{
+    // The command is only valid in guided mode.
```

Comment:
Should we also check were in fixed wing flight (ie not VTOL) and that we are currently flying?

Reply:
I now check quadplane is assisting. Can you give it a look?